lib/net/ssh/transport/algorithms.rb

```diff
@@ -48,9 +48,9 @@ class Algorithms
 
           encryption: %w[aes256-ctr aes192-ctr aes128-ctr aes256-gcm@openssh.com aes128-gcm@openssh.com],
 
-          hmac: %w[hmac-sha2-512-etm@openssh.com hmac-sha2-256-etm@openssh.com
-                   hmac-sha2-512 hmac-sha2-256
-                   hmac-sha1]
+          hmac: %w[hmac-sha1
+                   hmac-sha2-512-etm@openssh.com hmac-sha2-256-etm@openssh.com
+                   hmac-sha2-512 hmac-sha2-256]
```

Comment:
Wouldn't it be safer to have non-ETM go before ETM variants if ETM is not quite working as we expect?